product_docs/docs/pgd/5/ddl/ddl-locking.mdx

```diff
@@ -63,19 +63,17 @@ normally doesn't take an EXCLUSIVE LOCK or higher.
 
 Waiting for pending DML operations to drain can take a long time and even longer
 if replication is currently lagging.
-This means that schema changes affecting row representation and constraints,
-unlike with data changes, can be performed only while all configured nodes
+This means that, unlike with data changes, schema changes affecting row representation and constraints can be performed only while all configured nodes
 can be reached and are keeping up reasonably well with the current write rate.
 If such DDL commands must be performed while a node is down, first remove the
 down node from the configuration.
 
-
-**All** eligible data notes must agree to grant a global DML lock before the lock is granted.
+All eligible data notes must agree to grant a global DML lock before the lock is granted.
```

Comment:
I imagine you want to emphasize this, but we don't have a standard for emphasis (I don't even think DITA has a tag for that), and I don't think it's necessary. Typically, if I want something to really stand out, I'll put it in a note. Otherwise, the weight of the words themselves (I think "all" in itself is pretty emphatic) should do it.

Reply:
```suggestion
All eligible data nodes must agree to grant a global DML lock before the lock is granted.
```

Reply:
Ah, I missed the typo!